clients/src/main/java/org/apache/kafka/common/network/TransportLayer.java

```diff
@@ -113,4 +114,10 @@
      * @see FileChannel#transferTo(long, long, java.nio.channels.WritableByteChannel)
      */
     long transferFrom(FileChannel fileChannel, long position, long count) throws IOException;
+
+    /**
+     * Get information about the SSL connection that this transport layer is using, or None if this
+     * transport layer is not using SSL.
+     */
```

Comment:
`None` is not a Java thing, right? Also, wouldn't this also return an empty option if it was SSL but the handshake had not been completed?

Reply:
fixed